internal/plugins/golang/init.go

```diff
@@ -51,3 +59,86 @@ func (p *initPlugin) Run() error {
 
 	return nil
 }
+
+const (
+	makefileBundleVarFragment = `# Current operator version
+VERSION ?= 0.0.1
+# Bundle image URL
+BUNDLE_IMG ?= controller-bundle:$(VERSION)`
+
+	makefileManifestsName = "manifests"
+	//nolint:lll
+	makefileManifestsFragment = `manifests: controller-gen
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	operator-sdk generate bundle -q --kustomize
+`
+
+	makefileBundleName     = "bundle"
+	makefileBundleFragment = `# Generate a bundle directory
+bundle: manifests
+	kustomize build config/bundle | operator-sdk generate bundle -q --manifests --version $(VERSION)
+`
+
+	makefileBuildBundleName = "bundle-build"
+	//nolint:lll
+	makefileBuildBundleFragment = `# Build the bundle OCI image
+bundle-build: manifests
+	kustomize build config/bundle | operator-sdk generate bundle -q --manifests --metadata --overwrite --version $(VERSION)
+	operator-sdk bundle validate config/bundle
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
```

Comment:
Why we are changing the make file?
I do not think that we should impl the commands on it since we need to keep the operator-sdk commands ones since the make is not supported in win.

Reply:
To align with how kubebuilder implements functionality outside the core scaffold commands. Instead of having to run a bunch of intermediate steps to build a bundle image, a recipe like `bundle-build` captures all commands required to build that image in one `make` invocation.

Reply:
However, will not we have the sdk cli command for that? So, why change the makefile? 
Shows that the make do not work for win SO, so shows that we should not really relay on that.

Reply:
The SDK's CLI is changing to accommodate `kustomize` usage, implying that most manifests in `config/` are not the "end" manifest like those the SDK scaffolds in `deploy/`. We could wrap `kustomize` in `operator-sdk`, but that isn't the right call order in my opinion. We should be sticking with kubebuilder's `make` flow.

The makefile doesn't work for Windows?

Reply:
There was some debate with @shawn-hurley about keeping things like `operator-sdk build` around to accommodate Windows users who don't have `make`. But it does look like make can be installed natively in Windows (see [here](https://superuser.com/questions/808807/using-make-from-windows-powershell)), so maybe that fear was unfounded.

In any case, I second the Makefile approach. Our CLI has been too opinionated in some cases, and I think this approach (simple ready-made recipes in the Makefile + simple, do-one-thing SDK subcommands) strikes a nice balance.

Reply:
Agreed.